src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -142,6 +141,9 @@
 
     /**
      * TCP/IP port that the server is listening on.
+     *
+     * Before the first start, it will be 0. Once started, it is set to the actual port Jenkins is listening to.
+     *
```

Comment:
```suggestion
     * <p>
     * Before the first start, it will be 0. Once started, it is set to the actual port Jenkins is listening to.
     * <p>
```